frontend/src/components/taskSelection/action.js

```diff
@@ -304,12 +344,40 @@ export function TaskMapAction({ project, projectIsReady, tasks, activeTasks, act
                   </>
                 )}
                 {activeSection === 'history' && (
-                  <TaskHistory
-                    projectId={project.projectId}
-                    taskId={tasksIds[0]}
-                    commentPayload={taskHistory}
-                    mapperLevel={userDetails['mappingLevel']}
-                  />
+                  <>
+                    {activeTasks.length === 1 && (
+                      <>
+                        <TaskHistory
+                          projectId={project.projectId}
+                          taskId={tasksIds[0]}
+                          commentPayload={taskHistory}
+                          mapperLevel={userDetails['mappingLevel']}
+                        />
+                      </>
+                    )}
+                    {action === 'VALIDATION' && activeTasks.length > 1 && (
+                      <Accordion allowMultipleExpanded allowZeroExpanded onChange={handleFetch}>
+                        {activeTasks.map((t) => (
+                          <AccordionItem key={t.taskId} uuid={t.taskId}>
+                            <AccordionItemHeading className="b ttu tracked">
+                              <AccordionItemButton>
+                                <FormattedMessage {...messages.task} />
```

Comment:
It's better to use `<FormattedMessage {...messages.taskActivity} values={{n: t.taskId}} />`